js/react/components/auth/signup/SignUp.jsx

```diff
@@ -25,7 +25,7 @@ function SignUp({ modalRef, dispatch }) {
 					</a>
 				</div>
 			}
-			footer={<p>Agency partner users must sign up with work email</p>}
+			footer={<p>Agency partner users must sign up with work email.</p>}
```

Comment:
```suggestion
			footer={<p>Agency partners must log in with a work email.</p>}
```

Goes with [this suggestion](https://github.com/donategifts/donategifts/pull/654/files#r1304626617).

Reply:
@nick-w-nick Since this PR was approved a while ago, I will merge this and fix the typo in the new branch!